src/firebase/firebaseConfig.js

```diff
@@ -0,0 +1,16 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
```

Comment:
Theres an api key in this const variable. Can you put it in a .env file or config file to hide it ? Don't want to have our api key used for malicious intent.

Reply:
thank you for providing this link. https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public